src/internal/components/screenreader-only/index.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import styles from './styles.css.js';
+
+type ScreenreaderOnlyProps = Exclude<React.HTMLAttributes<HTMLDivElement>, 'className'>;
+
+/**
+ * Makes content now shown on a screen but still announced by screen-reader users.
+ * The component is suitable when the aria-label cannot be used, e.g. to avoid elemnts being announced as "blank".
+ *
+ * To exclude screenreader-only content for testing use `wrapper.find(':not([data-screenreader-only])')`.
```

Comment:
Not sure how to use this guideline, could you give a more detailed example?

Reply:
Example: https://github.com/cloudscape-design/components/pull/351/files#diff-5f254303359ce4f81256cf4e5809f2736eba9f5262de9047982f02b8e7864a82R60

Reply:
There is a class name directly, which can be used

```js
import visuallyHiddenStyles from '~internal/components/screenreader-only/styles.css.js'

wrapper.findAll(':not(${visuallyHiddenStyles.root})')
```

The tactic is always the same – do not add extra data-attributes unless absolutely necessary

Reply:
Alright, I don't mind it.